app/pages/sections/catalogue-solutions/order-item/contextCreator.js

```diff
@@ -31,12 +31,17 @@ const generateAddPriceTable = ({
   addPriceTable, price, itemUnitDescription, timeUnitDescription = '', errorMap,
 }) => {
   const columns = [];
+  let priceData = price;
+
+  if ((price || {}).toString().includes('.') && price.toString().split('.')[1].length < 2) {
+    priceData = parseFloat(price).toFixed(2);
+  }
```

Comment:
move this to a helper function within priceFormatter that can be unit tested. They are few combinations we want to test that I don't think we have covered. For example if the number is a whole number it shouldn't add the .00 to the end.